index.css

```diff
@@ -0,0 +1,78 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+.header1 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: auto;
+  padding: 30px 0;
+}
+
+.header1 h3 {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.header1 h3 a {
+  text-decoration: none;
+}
+
+.section1 {
+  background: url(./img//background1.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 90%;
+  margin: auto;
+  padding: 100px 0;
+}
```

Comment:
- [Optional] Consider styling this section with CSS flexbox as demanded by the project requirements.